lib/index.js

```diff
@@ -665,6 +665,20 @@ class BunnyBus extends EventEmitter{
         });
     }
 
+    _resubscribeAll(cb) {
+
+        const $S = $.subscriptions;
+        const subscribeFunctions = [];
+        for ( const [queueName, values] of $S._subscriptions ) {
+            if (!$S.isBlocked(queueName)) {
+                subscribeFunctions.push( $._subscribe.bind(this, queueName, values.handlers, values.options));
+                $S.clear(queueName);
```

Comment:
Just checking - this is just to clean things up nicely before re-subscribing?

Reply:
This is because the _subscribe method errors if a subscription already exists. I didn't want to remove it for fear of duplicate subscriptions.